.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,8 +44,8 @@ body:
           label: Steps to reproduce
           description: Tell us how to reproduce this issue and provide a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve), you can use the template below
           value: |
-              1. 
-              2. 
```

Comment:
I think these trailing newlines may be intentional

Reply:
Trailing spaces are intentional, so you start directly typing

Reply:
Will fix when I get back home